### ISSUE TITLE
XEP-0054: Setting vcard data

### ISSUE THREAD

**Issue by pulkomandy** [HaikuArchives/Renga#53]
It should be possible to set and edit own vcard.

**Comment by CodeforEvolution** [comment]
As a possible later feature, maybe one could be allowed to import/export Haiku People files to provide better system integration?

**Comment by pulkomandy** [comment]
Yes, I had that in mind too. But I'm not sure if the vcard is actually used much besides for avatars. And in fact the spec explicitly warns about sharing personal details there since you don't know who could access them. So we need at least a warning about that when importing here, and maybe filter out most fields by default.
